app/assets/javascripts/components/components/status_content.jsx

```diff
@@ -93,7 +93,7 @@ const StatusContent = React.createClass({
     const { status } = this.props;
     const { hidden } = this.state;
 
-    const content = { __html: emojify(status.get('content')) };
+    const content = { __html: emojify(status.get('content').replace(/\s*(<p>|<\/p>)\s*/g, '$1').replace(/\s{2}/g, ' &nbsp;')) };
```

Comment:
If your changes remove `<p>` tags, what is this [regex-for-html](http://stackoverflow.com/a/1732454) doing here?

You also seem to be replacing two spaces with one non-breaking space. I am not aware of non-breaking spaces having a different width to normal ones, so I am puzzled by this.